nglview/show.py

```diff
@@ -281,6 +286,29 @@ def show_schrodinger_structure(mol, **kwargs):
     return NGLWidget(structure_trajectory, **kwargs)
 
 
+def show_biopython(mol, **kwargs):
```

Comment:
Need to add unitttest for this function. 

https://github.com/arose/nglview/blob/master/nglview/tests/test_widget.py#L467